db/re/item_db.txt

```diff
@@ -756,7 +756,7 @@
 1412,Lance__,Lance,5,60000,,2500,185,,3,0,0x00004082,63,2,34,3,33,1,5,{},{},{}
 1413,Gungnir,Gungnir,5,20,,500,120,,3,0,0x00004082,63,2,2,4,4,1,4,{ bonus bAtkEle,Ele_Wind; bonus bPerfectHitRate,25; bonus bHit,30; },{},{}
 1414,Gelerdria,Gelerdria,5,20,,700,145,,3,0,0x00004082,63,2,2,4,48,1,4,{ bonus bAtkEle,Ele_Earth; bonus bMaxHP,800; bonus bMaxSP,-50; },{},{}
-1415,Skewer,Brocca,5,20,,850,100,,3,0,0x00004082,63,2,2,4,48,1,4,{ bonus bIgnoreDefClass,Class_Normal; bonus2 bAddEle,Ele_Neutral,25; },{},{}
+1415,Skewer,Brocca,5,20,,850,85,,3,0,0x00004082,63,2,2,4,48,1,4,{ bonus bIgnoreDefClass,Class_Normal; bonus2 bAddEle,Ele_Neutral,25; },{},{}
```

Comment:
I think you confused weight and attack here. Weight should be 85 (850 in database value) and attack should be 100.

Reply:
You're correct.
Thank you.